Mono.Cecil/AssemblyReader.cs

```diff
@@ -929,18 +929,14 @@ void AddNestedMapping (uint declaring, uint nested)
 			metadata.SetReverseNestedTypeMapping (nested, declaring);
 		}
 
-		static TValue [] AddMapping<TKey, TValue> (Dictionary<TKey, TValue []> cache, TKey key, TValue value)
+		static List<TValue> AddMapping<TKey, TValue> (Dictionary<TKey, List<TValue>> cache, TKey key, TValue value)
 		{
```

Comment:
This is the only change that isn't a simple `T[]` -> `List<T>` and `Length` -> `Count`.